common/MessageQueue.hpp

```diff
@@ -77,7 +77,7 @@ class MessageQueue
         return _queue.empty();
     }
 
-    bool size()
+    size_t size()
```

Comment:
failed to see bool vs size_t the last time, might as well suggest it could be const this time

Reply:
=)